src/main/java/com/google/codeu/servlets/HomeServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.codeu.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/home")
+public class HomeServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws IOException, ServletException {
+
```

Comment:
You'll need to add code here to filter out requests for URL's other than "/". Here is some sample code.

    if (request.getServletPath() != "/") {
      System.err.println("Invalid path requested:");
      System.err.println("\tpath " + request.getServletPath());
      System.err.println("\turl " + request.getRequestURL());
      response.sendError(HttpServletResponse.SC_NOT_FOUND);
      return;
    }

This will generate 404 pages for anything that gets sent to this servlet that isn't what we expect. Try requesting something like http://localhost:8080/foo to see what this does.